README.md

```
@@ -34,6 +34,8 @@ A private family platform for sharing letters, photos, and memories. Upload scan
 
 ## Live Demo
 
+** THIS REPO IS IN ACTIVE DEVELOPMENT AND WILL CHANGE OFTEN **
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)